### PR TITLE
Add IMG Tag, Fix RP Tag, Fix centered QR and IMG

### PR DESCRIPTION
**Pull request by deckyfx**
**What it is?**

- Add IMG tag so now it can print an image using command inside the design instead of using the API method,
  the syntax is ```{IMG[file://image/path/file.png]}```
- Fix the QR and IMG TAG so it can be mixed with other TAG in one line in design, eg:
  ```{QR[data]}{C}``` will print the QR Code centered
  ```{IMG[file://image/path/file.png]}{C}``` will print the Image centered
- Try to implement printer width 76
- Convert image to RGBA_565 during reading
- Fix the issue that causes can not have more than one same {RP:n:*} tag in one line
  eg: previously this will cause error ```{RP:5:-}Some Text{RP:5:-}``` 
  since the replacer routine will replace both RP tag, but then try to replace it again in next iteration
- Replace image resizer static method so it can retain aspect ratio while can have max-width or max-height
- Put some lint suppressor to prevent IDE to reporting error due to lack of Bluetooth permission in the manifest

**Suggestion:**

- Need a way to freely customize printed QR and IMG size

### CODE REVIEW

File: android/src/main/java/leesiongchan/reactnativeescpos/EscPosModule.java [hunk 1]
```diff
@@ -48,14 +51,15 @@
     public EscPosModule(ReactApplicationContext reactContext) {
         super(reactContext);
         this.reactContext = reactContext;
-
         scanManager = new ScanManager(reactContext, BluetoothAdapter.getDefaultAdapter());
+        System.out.println("Injected!");
```
Comment (leesiongchan) [inline android/src/main/java/leesiongchan/reactnativeescpos/EscPosModule.java:55]: Can you help remove this?

File: android/src/main/java/leesiongchan/reactnativeescpos/PrinterService.java [hunk 2]
```diff
@@ -93,14 +103,22 @@ public void printDesign(String text) throws IOException {
...
-    public void printImage(String filePath) throws IOException {
+    public Bitmap readImage(String filePath, ReactApplicationContext reactContext) throws IOException {
         Uri fileUri = Uri.parse(filePath);
-        Bitmap image = BitmapFactory.decodeFile(fileUri.getPath());
-        printImage(image);
+        Bitmap image = null;
+        BitmapFactory.Options op = new BitmapFactory.Options();
+        op.inPreferredConfig = Bitmap.Config.ARGB_8888;
+        image = BitmapFactory.decodeFile(fileUri.getPath(), op);
+        System.out.println(filePath);
```
Comment (leesiongchan) [inline android/src/main/java/leesiongchan/reactnativeescpos/PrinterService.java:112]: and here

**Comment by leesiongchan** [comment]
Can you also update your changes to the README.md

File: android/src/main/java/leesiongchan/reactnativeescpos/EscPosModule.java (hunk 1 above)
Comment (deckyfx) [reply android/src/main/java/leesiongchan/reactnativeescpos/EscPosModule.java:55]: ok my bad

File: android/src/main/java/leesiongchan/reactnativeescpos/PrinterService.java (hunk 2 above)
Comment (deckyfx) [reply android/src/main/java/leesiongchan/reactnativeescpos/PrinterService.java:112]: ok, to make sure, just the System.out?

**Comment by deckyfx** [comment]
> Can you also update your changes to the README.md

how should I write the MD?

File: android/src/main/java/leesiongchan/reactnativeescpos/PrinterService.java (hunk 2 above)
Comment (leesiongchan) [reply android/src/main/java/leesiongchan/reactnativeescpos/PrinterService.java:112]: yea haha, we dont want log in our lib

**Comment by leesiongchan** [comment]
@deckyfx thank you for your contribution!

**Pull request merged**